index.html

```diff
@@ -55,5 +59,157 @@ <h2 class="social">LET'S CONNECT</h2>
     </ul>
 </div>
 </section>
+<section id="About">
```

Comment:
- [ ] Please I highly recommend wrapping the main work section container with the CSS property `display: grid`  which implements mobile responsiveness, a better user experience per the project requirements. 😄